src/commtrack.js

```diff
@@ -191,10 +191,19 @@ define([
             }
         },
         transferMugValidation = function (mug) {
-            if (mug.p.dest.value || mug.p.src.value) {
-                return 'pass';
+            var error;
+            if ((mug.p.dest && mug.p.dest.value) || (mug.p.src && mug.p.src.value)) {
+                error = {key: "commtrack-transfer-src-dest-error"};
```

Comment:
A message object without a message (level is not used for anything right now) [will cause a previous error with the same key to be removed](https://github.com/dimagi/Vellum/blob/mug-messages/src/mugs.js#L356-L358). This is important because it causes the error to disappear when the property becomes valid (validation is triggered by property change, among other things).